tests/Cache/CacheTaggedCacheTest.php

```diff
@@ -112,7 +112,7 @@ public function testRedisCacheTagsPushStandardKeysCorrectly()
         $conn->shouldReceive('sadd')->once()->with('prefix:bar:standard_ref', 'prefix:'.sha1('foo|bar').':key1');
         $store->shouldReceive('push')->with(sha1('foo|bar').':key1', 'key1:value');
 
-        $redis->put('key1', 'key1:value');
+        $redis->put('key1', 'key1:value', 0);
```

Comment:
This highlights the BC break, no? I think this should come with an upgrade documentation note.

Reply:
I agree. Will make a note to add this to the upgrade guide. Thanks 👍